src/main/java/io/cdap/plugin/gcp/bigquery/sink/BigQuerySinkConfig.java

```diff
@@ -164,6 +173,21 @@ public String getDedupeBy() {
     return Strings.isNullOrEmpty(dedupeBy) ? null : dedupeBy;
   }
 
+  @Nullable
+  public String getPartitionFilter() {
```

Comment:
does this need to be public?

Reply:
@CuriousVini yes this needs to be public since it is used in `BigQuerySink.java`

```
if (config.getPartitionFilter() != null) {
      baseConfiguration.set(BigQueryConstants.CONFIG_PARTITION_FILTER, getConfig().getPartitionFilter());
    }
```